ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml

```diff
@@ -32,7 +34,32 @@ periodics:
       - name: JOB_NAME_SAFE
         value: e2e-operator-metrics
       - name: TEST_COMMAND
-        value: make e2e-operator-metrics
+        value: |
+          #!/bin/bash
+
+          git clone https://github.com/openshift/origin.git -b master
+          cd origin
+
+          export GO111MODULE=off
+
+          # install Kubectl required by the test.
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+          mkdir -p /tmp/shared
+          export PATH=$PATH:/tmp/shared
+          mv ./kubectl /tmp/shared/kubectl
+
+          # make and run test
+          make WHAT=cmd/openshift-tests
+
+          cd ..
+          DIR="./origin/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH)"
+
+          # OLM sanity tests
+          ${DIR}/openshift-tests run all --dry-run | grep "\[sig-operator\].*OLM" | ${DIR}/openshift-tests run --junit-dir=${ARTIFACT_DIR} -f -
+
+          # Install and uninstall rh-operators and run metric tests after
+          make e2e-operator-metrics || ${DIR}/openshift-tests run all --dry-run | grep "\[sig-instrumentation\].*Prometheus.*should report telemetry if a cloud.openshift.com token is present" | ${DIR}/openshift-tests run --junit-dir=${ARTIFACT_DIR} -f - || ${DIR}/openshift-tests run all --dry-run | grep "\[sig-instrumentation\].*Prometheus\|\[sig-instrumentation\].*Alerts"| grep "should have a Watchdog alert in firing state the entire cluster run\|should report telemetry if a cloud.openshift.com token is present\|should start and expose a secured proxy and unsecured metrics" | ${DIR}/openshift-tests run --junit-dir=${ARTIFACT_DIR} -f -
```

Comment:
what is the e2e-operator-metrics make target?

Reply:
This feels a bit hacky to me, is there no other way to do this? Do we have a precedence for this way of running tests? 🤔

Reply:
I can not say much about `origin`, but the make target is [1] and I don't see this as a hacky method since using make target is very common for a lot of the repositories testing in `release`. We can move the make target here but it is just a trivial one line change. 
[1] https://github.com/operator-framework/operator-lifecycle-manager/blob/2dea823abbacc286715254078d28b5058655362e/Makefile#L133